packages/commercetools/theme/components/MyAccount/NewsletterForm.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+    <form
+      id="newsletter-form"
+      class="form"
+      @submit.prevent="submitForm"
+    >
+      <p class="form__title">{{ $t('Sections that interest you') }}</p>
+      <div class="form__checkbox-group">
+        <SfCheckbox
+          v-model="newsletter.woman"
+          @change="updateNewsletterField('woman', newsletter.woman)"
```

Comment:
I don't think it will change anything. Should we use `!newsletter.woman` or `$event`'s value?